authorization/src/main/java/com/terabite/movement/controller/CategoryController.java

```diff
@@ -52,18 +59,26 @@ public ResponseEntity<List<Category>> getALlCategoriesByMovementId(@PathVariable
         return new ResponseEntity<>(categories, HttpStatus.OK);
     }
 
+    @GetMapping("/categories/{category_id}/movements")
```

Comment:
It seems like this returns a similar payload to `/categories/{category_id}` (a list of movements), so I would say keep one or the other; no need to have this redundancy.